hoomd/hpmc/tune/boxmc_move_size.py

```diff
@@ -0,0 +1,302 @@
+            reach the specified target.
+        types (list[str]): A list of string particle
+            types to tune the move size for, defaults to None which upon
+            attaching will tune all types in the system currently.
+        max_move_size (float): The maximum value of a volume move
+            size to attempt.
+
+    Warning:
+        Over-limiting the maximum move sizes can lead to the inability to
+        converge to the desired acceptance rate.
+    """
+    _internal_class = _InternalBoxMCMoveSize
+
+    @classmethod
+    def scale_solver(cls,
+                     trigger,
+                     boxmc,
+                     moves,
+                     target,
+                     max_move_size=None,
+                     max_scale=2.,
+                     gamma=1.,
+                     tol=1e-2):
+        """Create a `BoxMCMoveSize` tuner with a `hoomd.tune.ScaleSolver`.
+
+        Args:
+            trigger (hoomd.trigger.Trigger): ``Trigger`` to determine when to
+                run the tuner.
+            boxmc (hoomd.hpmc.update.BoxMC): The `hoomd.hpmc.update.BoxMC`
+                object to tune.
+            moves (list[str]): A list of types of moves to tune. Available
+                options are 'a' and 'd'.
+            target (float): The acceptance rate for trial moves that is desired.
+                The value should be between 0 and 1.
+            types (list[str]): A list of string particle types to tune the
+                move size for, defaults to None which upon attaching will tune
+                all types in the system currently.
+            max_move_size (float): The maximum value of a volume
+                move size to attempt.
+            max_scale (float): Maximum scale factor.
+            gamma (float): The value of gamma to pass through to
+                `hoomd.tune.ScaleSolver`. Controls the size of corrections to
+                the move size (larger values increase stability while increasing
+                convergence time).
+            tol (float): The absolute tolerance to allow between the current
+                acceptance rate and the target before the move sizes are
+                considered tuned. The tolerance should not be too much lower
+                than the default of 0.01 as acceptance rates can vary
+                significantly at typical tuning rates.
+        """
+        solver = ScaleSolver(max_scale, gamma, 'negative', tol)
+        return cls(trigger, boxmc, moves, target, solver, max_move_size)
+
+    @classmethod
+    def secant_solver(cls,
+                      trigger,
+                      boxmc,
+                      moves,
+                      target,
+                      max_move_size=None,
+                      gamma=0.8,
+                      tol=1e-2):
+        """Create a `BoxMCMoveSize` tuner with a `hoomd.tune.SecantSolver`.
+
+        This solver can be faster than `hoomd.tune.ScaleSolver`, but depending
```

Comment:
Correct this documentation, it appears to be copied and pasted from the mc integrator move size tuner.